### PR TITLE
Add ~1 year expiry to OP_RETURN protection of REQ-6-1

### PR DESCRIPTION
**Pull request by ftrader**
Based on estimated height of 478,061 for UASF activation, adding 365 * 144 blocks and rounding down to 530,000 .

### CODE REVIEW

**Comment by dgenr8** [comment]
ack

**Comment by deadalnix** [comment]
I'm good with that.

File: BUIP-HF/buip-hf-technical-spec.md [hunk 1]
```diff
@@ -127,14 +127,20 @@ specified by the user (default: 8MB) will take effect.
...
-with a specific magic data value shall be considered invalid.
+with a specific magic data value shall be considered invalid until
+block 530,000 inclusive.
 
 RATIONALE: To give users on the legacy chain (or other fork chains)
 an opt-in way to exclude their transactions from processing on the BUIP-HF
-fork chain.
+fork chain. The sunset clause block height is calculated as approximately
+1 year after currently planned UASF activation time (Aug 1 2017 00:00:00 GMT),
+rounded to an easier to remember number.
```
Comment (dagurval) [inline BUIP-HF/buip-hf-technical-spec.md]: rounded _down_ to

Comment (dagurval) [inline BUIP-HF/buip-hf-technical-spec.md]: how about changing "easier to remember" to "human friendly"?

Comment (ftrader) [inline BUIP-HF/buip-hf-technical-spec.md]: good suggestions imo, I will revise

**Comment by deadalnix** [comment]
Good.

**Pull request merged**